rpc/lib/server/http_server.go

```diff
@@ -18,9 +18,23 @@ import (
 	types "github.com/tendermint/tendermint/rpc/lib/types"
 )
 
-// Config is an RPC server configuration.
+// Config is a RPC server configuration.
 type Config struct {
+	// see netutil.LimitListener
 	MaxOpenConnections int
+	// mirrors http.Server#ReadTimeout
+	ReadTimeout time.Duration
+	// mirrors http.Server#WriteTimeout
+	WriteTimeout time.Duration
+}
+
+// DefaultConfig returns a default configuration.
+func DefaultConfig() *Config {
+	return &Config{
+		MaxOpenConnections: 0, // unlimited
+		ReadTimeout:        3 * time.Second,
+		WriteTimeout:       3 * time.Second,
```

Comment:
This default should be larger than the default for `TimeoutBroadcastTxCommit` which is 10s.
See: https://github.com/tendermint/tendermint/blob/4162ebe8b586deccc0e7476d8abafb75138bfe58/config/config.go#L357-L357